man/udevadm.xml

```diff
@@ -175,6 +175,13 @@
             <para>Cleanup the udev database.</para>
           </listitem>
         </varlistentry>
+        <varlistentry>
+          <term><option>--protected-tag=<replaceable>TAG</replaceable></option></term>
+          <listitem>
+            <para>Do not remove udev database for tagged devices. This takes effects only when
```

Comment:
I think this should say "This is only allowed when ..." because an error is thrown in other cases.